fhir-test/src/example.js

```diff
@@ -0,0 +1,23 @@
+const functionFromProcessor = async (userContext, events, done) => {
+  userContext.vars.requestBodyFromProcessor = "something";
+  // continue with executing the scenario:
+  return done();
+};
+
+// https://www.artillery.io/docs/guides/guides/http-reference#beforerequest-hooks
+function setJSONBody(requestParams, context, ee, next) {
+  // ...
```

Comment:
lefttover?

Reply:
no, just a placeholder for code whoever takes the example as a reference,